index.html

```diff
@@ -252,9 +284,10 @@ <h3>Defining Zome Functions</h3>
 
         <section>
           <fragment language="markdown" animate="by-line with-ancestry">
-            ### Defining Entries
+            ### Defining An Entry Format
```

Comment:
Just FYI, the term that the scaffolding tool and the developers docs have for this is "Entry Type".

Reply:
Fantastic-- @mmeigooni my local repo is in some funky state I can't figure out quickly. I'll try to do a deep debug soon, but until then would you please resolve any issues that we complete discussions on and merge?

Reply:
handled